src/components/views/rooms/RoomList2.tsx

```diff
@@ -17,13 +17,16 @@ limitations under the License.
 */
 
 import * as React from "react";
+import { Dispatcher } from "flux";
+import { Room } from "matrix-js-sdk/src/models/room";
+
```

Comment:
random gap is random

Reply:
we quite often have a gap between absolute and relative imports?

Reply:
confirmed intentional import divide